djangocms_page_meta/models.py

```diff
@@ -64,12 +64,9 @@ class PageMeta(PageExtension):
     twitter_type = models.CharField(
         _('Resource type'), max_length=255, choices=meta_settings.TWITTER_TYPES, blank=True
     )
-    gplus_author = models.CharField(
-        _('Author Google+ URL'), max_length=255, default='', blank=True,
-        help_text=_('Use the Google+ Name (together with "+")')
-    )
-    gplus_type = models.CharField(
-        _('Resource type'), max_length=255, choices=meta_settings.GPLUS_TYPES, blank=True,
+
+    schemaorg_type = models.CharField(
+        _('Resource type'), max_length=255, blank=True,
```

Comment:
I think this should point to `meta_settings.SCHEMAORG_TYPES`: it's too hard for editors to pick the correct type, and a properly crafted list must be prepared by the project developers